codespell_lib/data/dictionary.txt

```diff
@@ -45286,8 +45286,17 @@ sabatoshing->sabotaging
 sabatour->saboteur
 sacalar->scalar
 sacalars->scalars
+sacale->scale
+sacaled->scaled
+sacales->scales
+sacaling->scaling
 sacarin->saccharin
-sacle->scale
+saclar->scalar
```

Comment:
Any typos of `sacral` and `secular`?

Reply:
I'll add `sacral`. I couldn't find `secular`.